modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/manager/FeatureFlagPreferencesManager.java

```diff
@@ -68,8 +110,33 @@ private PortalPreferences _getPortalPreferences(long companyId) {
 		return portalPreferencesWrapper.getPortalPreferencesImpl();
 	}
 
+	private void _notifyCluster(long companyId, String key, boolean enabled) {
+		if (!_clusterExecutor.isEnabled()) {
+			return;
+		}
+
+		MethodHandler methodHandler = new MethodHandler(
+			_onNotifyMethodKey, companyId, key, enabled);
+
+		ClusterRequest clusterRequest = ClusterRequest.createMulticastRequest(
+			methodHandler, true);
+
+		clusterRequest.setFireAndForget(true);
+
+		_clusterExecutor.execute(clusterRequest);
+	}
+
 	private static final String _NAMESPACE = FeatureFlagConstants.FEATURE_FLAG;
 
+	private static final MethodKey _onNotifyMethodKey = new MethodKey(
```

Comment:
@W2jx Thanks a lot! Could you also change this field name to match the actual method? Thanks!